### PR TITLE
adding a new plate purpose for multiplexed sequencing submissions, to avoid FC

### PR DESCRIPTION
**Pull request by sabrine33**

### CODE REVIEW

**Review by dasunpubudumal — APPROVED** [review]

File: config/default_records/plate_purposes/005_limber_purposes.yml [hunk 1]
```diff
@@ -77,3 +77,7 @@ LTN Lib PCR XP:
 RVI Lib PCR XP:
   type: PlatePurpose
   stock_plate: false
+
+# A new plate purpose for multiplexed sequencing submissions,
+# avoiding FC on library quant submission.
+Library Stock Plate: *limber_input_96
```
Comment (KatyTaylor) [inline config/default_records/plate_purposes/005_limber_purposes.yml:83]: what does FC stand for?

Comment (sabrine33) [reply config/default_records/plate_purposes/005_limber_purposes.yml:83]: Fake Cherrypicking - It is used on the Confluence page, so I thought it was a common abbreviation—maybe not after all

Comment (KatyTaylor) [reply config/default_records/plate_purposes/005_limber_purposes.yml:83]: ah, fake cherrypick. I think a better description might be something like:

```
# A plate provided as ready-made libraries.
# These used to be imported as Stock Plates and then 'fake cherrypicked' onto 'LB-Lib-PCR-XP', which were allowed down the sequencing pipeline.
# Now, they can be imported and submitted for sequencing as this more accurate plate purpose.
```

**Comment by sabrine33** [comment]
closing this - as merging with develop went messy

**Pull request closed without merging**